### PR TITLE
Create empty file for dnsmasq.conf file

### PR DESCRIPTION
**Pull request by maximsava12**
As a pull request owner and reviewers, I checked that:
- [ x] Appropriate testing is done and actually running
- [ ] Appropriate documentation exists and/or is up-to-date:
  - [ ] README in the role
  - [ ] Content of the docs/source is reflecting the changes

### CODE REVIEW

**Comment by lewisdenny** [comment]
Hi @maximsava12 

I'm not sure you want `c3e9bf6b20ca5bc57edcf4f9104e30770508e11d` in this PR, please squash the other two commits. 

Also please explain in the commit message/description why this change is required, what issue are you seeing?

**Comment by pablintino** [comment]
> Hi @maximsava12
> 
> I'm not sure you want `c3e9bf6b20ca5bc57edcf4f9104e30770508e11d` in this PR, please squash the other two commits.
> 
> Also please explain in the commit message/description why this change is required, what issue are you seeing?

Hi @maximsava12, as @lewisdenny said, we need to understand why is this needed and in which circumstances this missing file is an issue.
Anyway, I'm not sure where that path comes from, cause the two (yes, crc uses two paths, depending on its version, as not that old versions used a dnsmasq container with a config file in a non-default path) we know about are: https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci/playbooks/multinode-customizations.yml#L74

File: roles/reproducer/tasks/configure_crc.yml [hunk 1]
```diff
@@ -32,6 +32,19 @@
         dns4_ignore_auto: true
         state: present
 
+    - name: Create dir for dnsmasq.conf
+      ansible.builtin.file:
```
Comment (cjeanner) [inline roles/reproducer/tasks/configure_crc.yml:36]: it's supposed to be a symlink pointing to /srv - do we really need this one? Care to provide some more context about the issue you're trying to solve?

**Comment by maximsava12** [comment]
Hi,
The context is 3-node Deployment failed with missing dnsmasq.conf file on stable branch.

**Comment by cjeanner** [comment]
@maximsava12 ok, so if consuming `stable` branch, please ensure you're using "older" CRC release (2.30 IIRC) so that it will follow the right path.

Or consume `main` branch instead - there's a partch in order to ensure dnsmasq is properly configured. The one you're proposing here will just blow the whole thing in `main`.

**Pull request closed without merging**